lms/static/js/dashboard/views/course_view.js

```diff
@@ -0,0 +1,147 @@
+;(function (define) {
+    'use strict';
+
+    define(['jquery', 'backbone', 'underscore', 'logger'],
+        function ($, Backbone, _, Logger) {
+            var CourseView = Backbone.View.extend({
+
+                events: {
+                    'click .action-more': 'toggleCourseActionsDropdown',
+                    'click .action-unenroll': 'unEnroll',
+                    'click .action-email-settings': 'emailSettings',
+                    'click #upgrade-to-verified': 'upgradeToVerified',
+                    'click #block-course-msg a[rel="leanModal"]': 'unRegisterBlockCourse'
+                },
+
+                initialize: function (options) {
+                    this.setElement(this.el);
+                    this.listenTo(options.tabbedView, 'rendered', this.rendered);
+                },
+
+                rendered: function () {
+                    var $actionUnroll = this.$('.action-unenroll');
+                    var $unRegisterBlockCourse = this.$('#unregister_block_course');
```

Comment:
You should only have 1 ```var``` per function, you can comma separate multiple var declarations.